static/app/components/events/breadcrumbs/breadcrumbsDataSection.tsx

```diff
@@ -150,47 +151,53 @@ export default function BreadcrumbsDataSection({
-          {t('Breadcrumbs')}
-        </GuideAnchor>
-      }
-      data-test-id="breadcrumbs-data-section"
-      actions={actions}
-    >
-      <ErrorBoundary mini message={t('There was an error loading the event breadcrumbs')}>
-        <div ref={setContainer}>
-          <BreadcrumbsTimeline
-            breadcrumbs={summaryCrumbs}
-            startTimeString={startTimeString}
-            // We want the timeline to appear connected to the 'View All' button
-            showLastLine={hasViewAll}
-            fullyExpanded={false}
-            containerElement={container}
-          />
-        </div>
-        {hasViewAll && (
-          <ViewAllContainer>
-            <VerticalEllipsis />
-            <div>
-              <ViewAllButton
-                size="sm"
-                // Since we've disabled the button as an 'outside click' for the drawer we can change
-                // the operation based on the drawer state.
-                onClick={() => (isDrawerOpen ? closeDrawer() : onViewAllBreadcrumbs())}
-                aria-label={t('View All Breadcrumbs')}
-                ref={viewAllButtonRef}
-              >
-                {t('View All')}
-              </ViewAllButton>
-            </div>
-          </ViewAllContainer>
-        )}
-      </ErrorBoundary>
-    </InterimSection>
+    <Fragment>
+      <InterimSection
+        key="breadcrumbs"
+        type={SectionKey.BREADCRUMBS}
+        title={
+          <GuideAnchor target="breadcrumbs" position="top">
+            {t('Breadcrumbs')}
+          </GuideAnchor>
+        }
+        data-test-id="breadcrumbs-data-section"
+        actions={actions}
+      >
+        <ErrorBoundary
+          mini
+          message={t('There was an error loading the event breadcrumbs')}
+        >
+          <div ref={setContainer}>
+            <BreadcrumbsTimeline
+              breadcrumbs={summaryCrumbs}
+              startTimeString={startTimeString}
+              // We want the timeline to appear connected to the 'View All' button
+              showLastLine={hasViewAll}
+              fullyExpanded={false}
+              containerElement={container}
+            />
+          </div>
+          {hasViewAll && (
+            <ViewAllContainer>
+              <VerticalEllipsis />
+              <div>
+                <ViewAllButton
+                  size="sm"
+                  // Since we've disabled the button as an 'outside click' for the drawer we can change
+                  // the operation based on the drawer state.
+                  onClick={() => (isDrawerOpen ? closeDrawer() : onViewAllBreadcrumbs())}
+                  aria-label={t('View All Breadcrumbs')}
+                  ref={viewAllButtonRef}
+                >
+                  {t('View All')}
+                </ViewAllButton>
+              </div>
+            </ViewAllContainer>
+          )}
+        </ErrorBoundary>
+      </InterimSection>
+      <LogsIssuesSection />
```

Comment:
This can belong in the breadcrumbs section or be it's own section, but since it should appear alongside and eventually replace breadcrumbs we should probably co-locate them.